keras/backend/tensorflow/core.py

```diff
@@ -91,6 +115,17 @@ def convert_to_tensor(x, dtype=None, sparse=True):
         return x
 
 
+def convert_to_dtensor(value, tensor_layout):
```

Comment:
I think this is actually the body of distribute_tensor, let's move it there.

Reply:
Done